mesheryctl/pkg/utils/spreadsheet_update.go

```diff
@@ -195,6 +246,41 @@ func appendSheet(srv *sheets.Service, cred, sheetId, appendRange string, values
 	return nil
 }
 
+func appendToCSV(csvPath string, values [][]interface{}) error {
+	// Open the CSV file in append mode, create if it doesn't exist
+	file, err := os.OpenFile(csvPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	// Create a new CSV writer
+	writer := csv.NewWriter(file)
+
+	// Write each row to the CSV
+	for _, row := range values {
+		stringRow := make([]string, len(row))
+		for i, val := range row {
+			stringRow[i] = fmt.Sprintf("%v", val) // Convert each value to a string
+		}
+
+		err := writer.Write(stringRow)
+		if err != nil {
+			return err
+		}
+	}
+
+	// Flush the writer to ensure all data is written to the file
+	writer.Flush()
+
+	// Check for any error during flush
+	if err := writer.Error(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
```

Comment:
Why do we need this we should be able to use the append to components sheet.

Reply:
> Why do we need this we should be able to use the append to components sheet.

This will make the function really messy. I'm trying to keep it simple
See the parameter it receives
```
srv *sheets.Service, cred, sheetId, appendRange string, values [][]interface{}
```

Also append the sheet logic uses srv package to update a sheet and that's a differnt logic for the csv, since it is local
https://github.com/meshery/meshery/pull/12051#discussion_r1763345173

Reply:
You are right but in future we would be migrating everything to meshkit so that server can use it too and users can import models using the csv from UI itself. Please reconsider using a single function but if you are unable to figure out an efficient way of handling this inside the same function we can go ahead with 2 different functions. Just a point to be kept in mind is we would be migrating it to meshkit so yes please consider that while writing this command.
Hope the suggestion helps. :sweat_smile: